charts/hoprd-operator/templates/crd.yaml

```diff
@@ -32,11 +32,17 @@ spec:
             spec: # There is only one (required) field named "replicas" specifying how many pods are created by the Operator
               type: object
               properties:
-                environment:
+                environmentName:
```

Comment:
You could nest this:
```yaml
environment:
  type: ...
  name: ...
```

Reply:
In a future PR, I will do a reorganization of all the properties. I have in mind to differenciate between `Kubernetes properties` and `hoprd arguments`. Something like
`````
deployment:
  resources: ...
  securityContext: ....
args:
   environment:
     name: monte_rosa
     type: production
   version: riga
  announce: true

````